frontend/src/components/application/OpportunityCard.tsx

```diff
@@ -0,0 +1,123 @@
+import { useTranslations } from "next-intl";
+import { Grid, GridContainer } from "@trussworks/react-uswds";
+
+interface OpportunityOverview {
+  name?: string | null;
```

Comment:
I'm assuming all of these props are coming from an opportunity API response object, in which case we can likely just use the response object itself, either the full opportunity or just the summary depending on what you need:

https://github.com/HHS/simpler-grants-gov/blob/main/frontend/src/types/opportunity/opportunityResponseTypes.ts